lib/takeaway.rb

```diff
@@ -0,0 +1,57 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Takeaway
+
+  attr_reader :current_order, :order
+
+  def initialize(menu = Menu.new, twilio)
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)